src/WebJobs.Script/Description/Binding/ApiHubBindingMetadata.cs

```diff
@@ -35,6 +35,22 @@ public override void ApplyToConfig(JobHostConfigurationBuilder configBuilder)
                 connectionString = Utility.GetAppSettingOrEnvironmentValue(Connection);
             }
 
+            FileTraceWriter traceWriter = null;
```

Comment:
I just checked in a change to add the TraceWriter to JobHostConfiguration. So if you pull and rebase, you can access via configBuilder.TraceWriter. Use that writer (it is a composite that includes the FileTraceWriter).